### PR TITLE
PS-8105 Release notes and release note tasks for 8.0.28-19 (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.7 version
modified conf.py
modified release-notes_index
added Percona-Server-8.0.28-19

### CODE REVIEW

File: doc/source/release-notes/Percona-Server-8.0.28-19.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+.. contents::
+   :local:
+
+Release Highlights
+=================================================
+
+The following lists a number of the bug fixes for *MySQL* 8.0.28, provided by Oracle, and included in Percona Server for MySQL:
+
+* Using the ``ASCII`` shortcut for ``CHARACTER SET latin1`` and ``UNICODE`` shortcut for ``CHARACTER SET ucs2`` are deprecated. Using either shortcut raises a warning to use ``CHARACTER SET`` instead. Expect that these shortcuts will be removed in a future version.
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.28-19.rst]: are deprecated -> is deprecated

File: doc/source/release-notes/Percona-Server-8.0.28-19.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* A stored function and a loadable function with the same name can share the same namespace. Add the schema name when invoking a stored function in the shared namespace. The server generates a warning when function names collide.
+* InnoDB supports ``ALTER TABLE ... RENAME COLUMN`` operations when using ``ALGORITH=INSTANT``. 
+* The limit for ``innodb_open_files`` includes temporary tablespace files. This addition is a change and the temporary tablespace files were not counted in previous versions.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.28 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-28.html>`__.
+
+Bugs Fixed
+=================================================
+
+* :jirabug:`PS-7999` The ``FEDERATED`` storage engine would not reconnect when a ``wait_timeout`` was exceeded. (Thanks to Sami Ahlroos for reporting this issue) (Upstream :mysqlbug:`105878`)
```
Comment (alina-derkach-oaza) [inline doc/source/release-notes/Percona-Server-8.0.28-19.rst]: :jirabug:`PS-7999` -> add a colon after the ticket number `PS-7999`:

File: doc/source/release-notes/Percona-Server-8.0.28-19.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.28-19.rst]: Rewritten to remove the "Using"

File: doc/source/release-notes/Percona-Server-8.0.28-19.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.28-19.rst]: added

**Comment by breisig** [comment]
Hi. Can we get this released really soon? MySQL 8.0.28 was released 2 months ago.

**Comment by breisig** [comment]
It looks like 8.0.28 has a type encoding/decoding issues for the DATE, TIME, DATETIME, and TIMESTAMP types using prepared statements. It looks like it's already fixed upstream for the 8.0.29 release. Can we please get this fixed in 8.0.28? [ https://bugs.mysql.com/bug.php?id=106352 ]

**Comment by breisig** [comment]
Any update on this release?

File: doc/source/_res/rn/useful-links.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Useful links
+========================
+
+- The installation instructions for `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/installation.html>`__ are available.
+
+- This is the location for the `Percona Server downloads <https://www.percona.com/downloads/>`__.
+
+- The GitHub location for `Percona Server for MySQL <https://github.com/percona/percona-server>`__
```
Comment (alina-derkach-oaza) [inline doc/source/_res/rn/useful-links.txt]: I would change it to smth like:
This is the location for `Percona Server for MySQL <https://github.com/percona/percona-server>`__  on GitHub.

File: doc/source/_res/rn/useful-links.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Useful links
+========================
+
+- The installation instructions for `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/installation.html>`__ are available.
```
Comment (alina-derkach-oaza) [inline doc/source/_res/rn/useful-links.txt]: I would change it to smth like:
This is the instruction on how to install `Percona Server for MySQL <https://www.percona.com/doc/percona-server/LATEST/installation.html>`__ .

File: doc/source/_res/rn/useful-links.txt (hunk 3 above)
Comment (patrickbirch) [reply doc/source/_res/rn/useful-links.txt]: Updated.

File: doc/source/_res/rn/useful-links.txt (hunk 4 above)
Comment (patrickbirch) [reply doc/source/_res/rn/useful-links.txt]: Updated.

**Review by alina-derkach-oaza — APPROVED** [review]

**Review by george-lorch — APPROVED** [review]

**Comment by breisig** [comment]
Did this get fixed in Percona? It looks like 8.0.28 has a type encoding/decoding issues for the DATE, TIME, DATETIME, and TIMESTAMP types using prepared statements. It looks like it's already fixed upstream for the 8.0.29 release. Can we please get this fixed before the release? [ https://bugs.mysql.com/bug.php?id=106352 ]

**Comment by breisig** [comment]
I think theirs something wrong with the latest percona-server-shared-8.0.28-19.1.el8.x86_64.rpm from the testing repo today. I get the following when trying to install the latest files from today. It says something about miscalculated checksum [Downloading successful, but checksum doesn't match. Calculated: 6d4c5b19fae27c1da859f19f9e2a72ba3806a3f8717bb87ddf0fc2a6cb3e78be(sha256)  Expected: 08f2b628f0e085b18ad3788db939a7415f51affde9bce050beaa913816206fc0(sha256)]

```
Dependencies resolved.
==================================================================================================================================================================================================================================
 Package                                                         Architecture                              Version                                              Repository                                                   Size
==================================================================================================================================================================================================================================
Installing:
 percona-server-devel                                            x86_64                                    8.0.28-19.1.el8                                      Percona_Yum_Repo                                    2.2 M
Installing dependencies:
 percona-server-shared                                           x86_64                                    8.0.28-19.1.el8                                      Percona_Yum_Repo                                    1.5 M
 percona-server-shared-compat                                    x86_64                                    8.0.28-19.1.el8                                      Percona_Yum_Repo                                    1.2 M

Transaction Summary
==================================================================================================================================================================================================================================
Install  3 Packages

Total size: 4.9 M
Total download size: 1.5 M
Installed size: 29 M
Is this ok [y/N]: y
Downloading Packages:
[SKIPPED] percona-server-devel-8.0.28-19.1.el8.x86_64.rpm: Already downloaded
[SKIPPED] percona-server-shared-compat-8.0.28-19.1.el8.x86_64.rpm: Already downloaded
[MIRROR] percona-server-shared-8.0.28-19.1.el8.x86_64.rpm: Downloading successful, but checksum doesn't match. Calculated: 6d4c5b19fae27c1da859f19f9e2a72ba3806a3f8717bb87ddf0fc2a6cb3e78be(sha256)  Expected: 08f2b628f0e085b18ad3788db939a7415f51affde9bce050beaa913816206fc0(sha256)
[MIRROR] percona-server-shared-8.0.28-19.1.el8.x86_64.rpm: Downloading successful, but checksum doesn't match. Calculated: 6d4c5b19fae27c1da859f19f9e2a72ba3806a3f8717bb87ddf0fc2a6cb3e78be(sha256)  Expected: 08f2b628f0e085b18ad3788db939a7415f51affde9bce050beaa913816206fc0(sha256)
[MIRROR] percona-server-shared-8.0.28-19.1.el8.x86_64.rpm: Downloading successful, but checksum doesn't match. Calculated: 6d4c5b19fae27c1da859f19f9e2a72ba3806a3f8717bb87ddf0fc2a6cb3e78be(sha256)  Expected: 08f2b628f0e085b18ad3788db939a7415f51affde9bce050beaa913816206fc0(sha256)
[MIRROR] percona-server-shared-8.0.28-19.1.el8.x86_64.rpm: Downloading successful, but checksum doesn't match. Calculated: 6d4c5b19fae27c1da859f19f9e2a72ba3806a3f8717bb87ddf0fc2a6cb3e78be(sha256)  Expected: 08f2b628f0e085b18ad3788db939a7415f51affde9bce050beaa913816206fc0(sha256)
[FAILED] percona-server-shared-8.0.28-19.1.el8.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success

The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'yum clean packages'.
Error: Error downloading packages:
  Cannot download percona-server-shared-8.0.28-19.1.el8.x86_64.rpm: All mirrors were tried
```

**Comment by patrickbirch** [comment]
Thank you for the comments. We do not monitor pull requests for feature requests or issues. You are welcome to comment in the Percona Community Forum - https://forums.percona.com/c/mysql-mariadb/percona-distribution-for-mysql/42

**Comment by breisig** [comment]
Thanks @patrickbirch , I  will take a look at it. It looks like MySQL 8.0.29 was just released.

**Comment by breisig** [comment]
Any idea on this 8.0.28 release? This release is not even out yet and yet you should start be working on releasing MySQL 8.0.29 within 30 days.

File: doc/source/release-notes/Percona-Server-8.0.28-19.rst [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+New Features
+=================================================
+
+* :jirabug:`PS-7586`: Adds a Key Management Interoperability Protocol (KMIP) component.
+
+Improvement
+=================================================
+
+* :jirabug:`PS-7871`: Using the ``SET_VAR`` syntax, MyRocks variables can be set dynamically.
+* :jirabug:`8064`: The ability to change log file locations dynamically is restricted.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.28-19.rst]: PS-8064

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-8.0.28-19.rst (hunk 5 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.28-19.rst]: Fixed.

**Pull request merged**